.github/workflows/build.yml

```
@@ -12,7 +12,9 @@ jobs:
         uses: actions/checkout@v3
       - name: Set imagemagick version
```

```suggestion
      - name: Set build variables
```

or something similar